client/src/components/HostViews/Respond.vue

```diff
@@ -98,7 +76,26 @@ export default {
       }, 350)
     })
   },
+  computed: {
+    timeRemaining() {
+      return this.$store.state.timeRemaining
+    },
+    wordDisplay() {
+      let wordsInPrompt = ''
+      for (let i = 0; i < this.words.length; i++) {
```

Comment:
Does JS have foreach loops? If so, here's a good use for one I believe.

Reply:
Also comment this up either way please.

Reply:
the reason I use a for loop is because I want to make sure the index before last gets an "and" added in. This is easier when I have an index number for i. I would have to do forEach(word => indexOf(word)) which is more awkward imo.

Reply:
Sounds good. Just add comments explaining what this is doing then please.

Reply:
will do!